elixir/apps/domain/lib/domain/change_logs/replication_connection.ex

```diff
@@ -56,9 +56,9 @@ defmodule Domain.ChangeLogs.ReplicationConnection do
         :ok
 
       {:error, %Ecto.Changeset{errors: errors} = changeset} ->
-        # Expected under normal operation when an account is deleted or we are catching up on
-        # already-processed but not acknowledged WAL data.
-        if foreign_key_error?(errors) or dupe_lsn_error?(errors) do
+        if Enum.any?(errors, &should_skip_change_log?/1) do
```

Comment:
I like this.  It does make it very clear what we are and aren't wanting to skip. 👍